backend/lib/res/default_config.json

```diff
@@ -55,7 +55,8 @@
         },
         "customizations": {
             "topicPrefix": "",
-            "provideMapData": true
+            "provideMapData": true,
+            "durationsAsInteger": false
```

Comment:
Does this apply to both homie and home assistant? If not, should we instead move it to `homie.durationsAsInteger`?